src/Symfony/Component/HttpKernel/Bundle/Bundle.php

```diff
@@ -204,4 +204,13 @@ protected function getContainerExtensionClass()
 
         return $this->getNamespace().'\\DependencyInjection\\'.$basename.'Extension';
     }
+
+    private function parseClassName()
+    {
+        $class = get_class($this);
```

Comment:
Since this is intended to target `master`, I'm guessing you could replace that by `static::class`?